bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -1911,7 +1909,8 @@ void BCStateTran::sourcePrepareBatch(uint64_t numBlocksRequested) {
   }
 
   if (invokeGetBlocks) {
-    if (clearContexts) {
+    // Contexts have to be cleared
```

Comment:
Rethinking, if you remove clearContexts, ca nyou please remove all the aboce commented code? lines 1883-1908.

Reply:
Removed, thanks.